src/firefly/js/visualize/ui/TriViewImageSection.jsx

```diff
@@ -86,7 +86,7 @@ export const makeMultiProductViewerTab= ({id,dataProductTableId}) => {
 };
 
 
-function BadgeLabel({labelStr}) {
+export function BadgeLabel({labelStr}) {
```

Comment:
since you are now exporting this component why don't you change the name to something more specific. Such as `ImagePinnedBadgeLabel`

Reply:
You're right. I had initially changed it to `ImageBadgeLabel` and then changed it back to `BadgeLabel` but then just imported it as `ImageBadgeLabel` (because there's already a `BadgeLabel` for charts as well). I think I'll change the Chart one's name to `ChartBadgeLabel` as well.